rgw/v2/tests/s3_swift/reusable.py

```diff
@@ -2226,15 +2226,34 @@ def put_bucket_website(rgw_conn, bucket_name):
         "ErrorDocument": {"Key": "error.html"},
         "IndexDocument": {"Suffix": "index.html"},
     }
-    put_bucket_wesite = rgw_conn.put_bucket_website(
+    put_bucket_website = rgw_conn.put_bucket_website(
         Bucket=bucket_name, WebsiteConfiguration=website_conf
     )
-    log.info(f"put_bucket_wesite {put_bucket_wesite}")
```

Comment:
@anrao19 please retain this log statement

Reply:
@hmaheswa , retained